source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -28,5 +28,76 @@ more naturally readable.
 Operations
 ----------
 
+Each of the following sections provides information and examples for
+aggregation expression operations available in the {+driver-short+}.
+This guide sorts these operations into categories based on general
+purpose and functionality.
+
+.. important::
+
+   The Query API expression that the driver generates from each
+   aggregation expression operation might not be
+   identical to the Query API expression provided in each example.
+   However, they are functionally equivalent.
+
+Boolean Expressions
+~~~~~~~~~~~~~~~~~~~
+
+You can invoke boolean operations on boolean values of type ``MqlBoolean``.
+
+The following table describes the boolean aggregation operation methods
+available in the {+driver-short+} and the equivalent expressions in the
+Query API:
```

Comment:
None of these methods have multiple signatures, but I can still call out that the driver includes multiple methods of the same name for some of these methods.

Reply:
> None of these methods have multiple signatures, but I can still call out that the driver includes multiple methods of the same name for some of these methods.

Cool, then no need to mention that if you're linking to the API docs since that should disambiguate them.